server/test/handlers/bookWorkerTimeslot.test.js

```diff
@@ -0,0 +1,47 @@
+const workerTimeslotHandler = require('../../models/handlers/workerTimeslot');
+
+describe('testing inputting of worker schedule functionality', () => {
```

Comment:
Hi Simran,
I noticed that for test case 1 and 2, the status of the workerTimeslotId = 5 and 7 must be unavailable. The same database is being used by multiple methods which edit the status of the entries in the worker_Timeslot table (eg.  S19-T2 changes the status from unavailable to available when an appointment is cancelled). Due to this, the value of the status parameter is constantly changing. Since your tests return true (indicating that the slot is unavailable), these tests may not return true if the value of status in the database for workerTimeslotId = 5 and 7 is unavailable.